### PR TITLE
Support ip creation

### PR DESCRIPTION
**Pull request by bobz965**
# Pull Request

- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

## What type of this PR

Examples of user facing changes:

- Features

https://github.com/kubeovn/kube-ovn/issues/3626

## Which issue(s) this PR fixes

Fixes #(issue-number)

## WHAT

copilot:summary

copilot:poem

## HOW

copilot:walkthrough

### CODE REVIEW

File: pkg/util/net.go [hunk 1]
```diff
@@ -286,16 +286,20 @@ func SplitIpsByProtocol(excludeIps []string) ([]string, []string) {
...
+	}
+	if IsValidIP(v6IP) {
+		ipList = append(ipList, v6IP)
+	}
+	if len(ipList) == 0 {
+		return ""
+	}
+	if len(ipList) == 1 {
+		return ipList[0]
+	}
```
Comment (oilbeater) [inline pkg/util/net.go:301]: strings.Join 可以处理元素个数为 0 和 1 的情况

File: pkg/controller/gc.go [hunk 2]
```diff
@@ -368,22 +367,26 @@ func (c *Controller) markAndCleanLSP() error {
...
+			klog.Infof("gc ip %s", ipCR.Name)
+			if err := c.config.KubeOvnClient.KubeovnV1().IPs().Delete(context.Background(), ipCR.Name, metav1.DeleteOptions{}); err != nil {
+				if k8serrors.IsNotFound(err) {
+					// ip cr not found, skip lsp gc
+					continue
+				}
+				klog.Errorf("failed to delete ip %s, %v", ipCR.Name, err)
+				return err
+			}
+			if ipCR.Spec.Subnet == "" {
```
Comment (oilbeater) [inline pkg/controller/gc.go:380]: 什么情况下 `spec.Subnet` 会是空

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: pkg/util/net.go (hunk 1 above)
Comment (bobz965) [reply pkg/util/net.go]: fixed

File: pkg/controller/gc.go (hunk 2 above)
Comment (bobz965) [reply pkg/controller/gc.go:380]: 尽管 webhook 已经限制了该 subnet 属性不能为空，但是如果不部署 webhook， spec 只可以直接修改的。所以忽略这种异常数据。

**Review by oilbeater — APPROVED** [review]

**Pull request merged**